pkg/kadm/groups.go

```diff
@@ -1460,7 +1460,12 @@ func (cl *Client) Lag(ctx context.Context, groups ...string) (DescribedGroupLags
 		}
 		if g.Err != nil {
 			delete(set, g.Group)
+			continue
 		}
+
+		// if no groups were specified for lag calculation, we will calculate lag for all groups
+		// adding groups to the set again.
```

Comment:
Can you change this to,
```
// If the input set of groups is empty, DescribeGroups returns all groups.
// We add to `set` here so that the Lag function itself can calculate
// lag for all groups.
```

The doc comment here was a bit confusing to me, so this took me a bit to realize what you're trying to do here.

Overall, good change though.

Reply:
Sorry for the delay, I have updated the doc comment.